app/messages.ts

```diff
@@ -98,6 +98,74 @@ export const messages = {
       },
     },
   },
+  householder: {
+    householderIdentityForm: {
+      validation: {
+        nameMinLength: 'نام سرپرست باید حداقل ۳ حرف باشد',
+        fatherNameMinLength: 'نام پدر باید حداقل ۳ حرف باشد',
+        nationalIdMinLength: 'شماره کارت ملی باید حداقل ۱۰ عدد باشد',
+        inValidNationalId: 'لطفا یک شماره کارت ملی معتبر وارد نمایید',
+        inValidSsn: 'لطفا یک شماره شناسنامه معتبر وارد نمایید',
```

Comment:
```suggestion
        invalidNationalId: 'لطفا یک شماره کارت ملی معتبر وارد نمایید',
        invalidSsn: 'لطفا یک شماره شناسنامه معتبر وارد نمایید',
```

the `invalid` should be written as whole